assets/src/domain/eventEditor/ui/datetimes/datesList/actionsMenu/dropdown/DateMainMenu.tsx

```diff
@@ -20,11 +20,11 @@ const DateMainMenu: React.FC<DateMainMenuProps> = ({ datetime }) => {
 	const title = trashed ? __('Permanently delete Datetime?') : __('Move Datetime to Trash?');
 	const message = trashed
 		? __(
-				'Are you sure you want to permanently delete this datetime? This action is permanent and can not be undone.'
-		  )
+			'Are you sure you want to permanently delete this datetime? This action is permanent and can not be undone.'
+		)
 		: __(
-				'Are you sure you want to move this datetime to the trash? You can "untrash" this datetime later if you need to.'
-		  );
+			'Are you sure you want to move this datetime to the trash? You can "untrash" this datetime later if you need to.'
+		);
```

Comment:
I don't understand why prettier is behaving this way, I didn't changed any configs locally